gr-blocks/grc/blocks_throttle2.block.yml

```diff
@@ -25,16 +25,16 @@ parameters:
     dtype: bool
     default: 'True'
     hide: ${ ('part' if str(ignoretag) == 'True' else 'none') }
--   id: duration_or_samples
-    label: Limited Quantity
+-   id: limit
+    label: Limit
     dtype: enum
-    options: ['time', 'items', 'neither']
-    option_labels: ["Max. Time", "Max. Items", "No Limit"]
+    options: ['auto', 'time', 'items']
+    option_labels: ["None", "Max Time (sec)", "Max Items"]
 -   id: maximum
-    label: 'Max. wait[s] or items'
+    label: 'Maximum'
     dtype: real
-    hide: ${ ('part' if duration_or_samples == 'neither' else 'none') }
-    default: 0.1
+    hide: ${ ('all' if limit == 'auto' else 'none') }
+    default: 1
```

Comment:
I can definitely live with the rest, but this new default of a 1s-chunkiness seems like an unwise change; the idea was that things like Qt Time sink look "smooth".

Reply:
I know you originally had 2 different values and I though they should be the same field :smile: 
OK, `0.1` is fine, since most people will care about time, not samples. There's really no right value for samples anyway.